.github/workflows/tests.yml

```
@@ -129,8 +129,8 @@ jobs:
               pytest --pyargs pywt
               python ../pywt/tests/test_doc.py
           elif [ "${REFGUIDE_CHECK}" == "1" ]; then
-              # Run doctests and check if the refguide contains entries from __all__
-              python util/refguide_check.py --doctests
+              # doctest docstrings
+              pytest --doctest-modules --pyargs pywt -v  --doctest-collect=api
               # Run Sphinx HTML docs builder, converting warnings to errors
               pip install -r util/readthedocs/requirements.txt
```

Looks like the path to the requirements file is incorrect, this should work:

```suggestion
              pip install -r ../util/readthedocs/requirements.txt
```